src/pivotal-ui/javascripts/pivotal-ui.js

```diff
@@ -7,3 +7,4 @@ require('pui-prismjs');
 require('./scale')();
 require('@npmcorp/pui-react-back-to-top/jquery-plugin');
 require('@npmcorp/pui-css-drop-down-menu');
+require('../components/forms');
```

Comment:
So, this reference is actually to the direct source file in a file that pulls everything in for the styleguide (this is the styleguide's extra js file).

In the case of when you'd `require` this outside of the styleguide, you'd require it as a module